storage/filesystem/storage_test.go

```diff
@@ -42,3 +45,9 @@ func (s *StorageSuite) TestFilesystem(c *C) {
 
 	c.Assert(storage.Filesystem(), Equals, fs)
 }
+
+func (s *StorageSuite) TestStructureShouldBeCreatedLazily(c *C) {
```

Comment:
The test name is misleading, what Structure?, and we are also not testing that contents are created lazily, just that the dir is created empty.

I suggest a new name for the test: `TestNewStorageShouldNotAddAnyContentsToDir`.